src/test/java/cmput402/tdd/AppTest.java

```diff
@@ -227,4 +228,257 @@ public void testDisplayPeople() {
         
     }
 
+    @Test
+    public void testEditRecipe(){
+        App app = new App();
+        Recipe recipe = new Recipe("apple pie");
+
+        Item item1 = new Item("apple",4);
+        Item item2 = new Item("sugar",2);
+
+
+        //Test 1: Add new item to recipe
```

Comment:
Is this test supposed to be a success scenario? If so we should fail in the catch instead of asserting something